recipes/zipstream-new/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "zipstream-new" %}
```

Comment:
@mr-c do you know why this is called `zipstream-new` and not `python-zipstream`?

Reply:
I used https://github.com/conda-incubator/grayskull to make this recipe based upon https://pypi.org/project/zipstream-new/ ; so the name must have come from there

Reply:
I see ok!